src/PipelineBuilderInterface.php

```diff
@@ -1,23 +1,17 @@
 <?php
+declare(strict_types=1);
+
 namespace League\Pipeline;
 
 interface PipelineBuilderInterface
 {
     /**
      * Add an stage.
-     *
-     * @param callable $stage
-     *
-     * @return $this
      */
-    public function add(callable $stage);
+    public function add(callable $stage): self;
```

Comment:
There's currently no way to express current instance returns in return types. This will effectively resolve as `PipelineBuilderInterface` which is clearer as a return type IMO. But the docblocks help editors to better reflect, since the return type can't.

Reply:
Updated.